### DISCUSSION TITLE
How to add "X"-style close button to window?

### DISCUSSION THREAD

**Discussion by ghost** [emilk/egui#1707] [Q&A]
I want to add a "X"-style close button to my window like in the web-demo. How can I do that?
My window is currently set up like this:
``` rust
egui::Window::new("test").show(&egui_ctx, |ui| {
        ui.label("test");
});
```
Thanks in advance!

**Comment by coderedart** [comment]
Please check docs first. Most of the simple things are covered already 

https://docs.rs/egui/latest/egui/containers/struct.Window.html#method.open

**Reply by ghost** [reply]
Wow I feel ashamed now. I have read and searched the docs but must have missed this. Thanks alot!
